demos/asp-net-core/Divergent.CompositionGateway/Program.cs

```diff
@@ -5,17 +5,14 @@ namespace Divergent.CompositionGateway
 {
     public class Program
     {
-        public static void Main(string[] args)
-        {
-            var host = new WebHostBuilder()
+        public static void Main(string[] args) =>
```

Comment:
This was created as the original VS/new project template, I don't think we should change it, it'll only raise time wasting questions